system/modules/playos-status.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+let
+  ttyNumber = 8;
+  tty = "tty${toString ttyNumber}";
+  ttyPath = "/dev/${tty}";
+  driverUrl = "http://127.0.0.1:8382";
+
+  script =
+    pkgs.writeShellScriptBin "print-status" ''
+      while :; do
+        printf "\033c"
```

Comment:
Maybe acquiring the info could be moved before the screen reset to remove/minimize the blank screen period?

Reply:
:+1: